File: data/data_transpiled/at_coder/agc002/A/4634429.java
public static void main ( String input ) {
  int a = Integer . parseInt ( input ) ;
  int b = Integer . parseInt ( input ) ;
  if ( a * b <= 0 ) {
    System . out . println ( "Zero" ) ;
  }
  else if ( a + b > 0 || ( a + b < 0 && ( a - b ) % 2 == 1 ) ) {
    System . out . println ( "Positive" ) ;
  }
  else {
    System . out . println ( "Negative" ) ;
  }
}

File: storage/data_original/at_coder/agc002/A/4634429.py
#AGC002 A 23:04 - 
'''
??
??a,b??????????a<=b????
????a,a+1,...,b????????????????0??????
???a,b?-10**9??10**9????a<=b

??
a*b<=0?????0
a*b>0??a+b>0??????
a*b>0??a+b<0??...
	a??b???????????????
		???a-b?????????
	a??b???????????????
		???a-b?????????
'''

a,b=map(int,input().split(' '))

if a*b<=0:
	print('Zero')
elif a+b>0 or (a+b<0 and (a-b)%2==1):
	print('Positive')
else:
	print('Negative')